x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -44,7 +42,7 @@ export const dataReducer: Reducer<DataState, ResolverAction> = (state = initialS
     };
     return nextState;
   } else if (action.type === 'appAbortedResolverDataRequest') {
-    if (treeFetcherParameters.equal(action.payload, state.tree.pendingRequestParameters)) {
+    if (treeFetcherParameters.equal(action.payload, state.tree?.pendingRequestParameters)) {
```

Comment:
:grey_question: So this is like how we know the tree (and its related events) are "ready" now? It's not really clear from reading this...

Reply:
let's discuss it